src/rhsm/config.py

```diff
@@ -199,20 +201,29 @@ def set(self, section, name, value):
                 self.add_section(section)
             super(RhsmConfigParser, self).set(section, name, value)
         if section == "logging" and name == "default_log_level":
-            self.is_log_level_valid(value)
+            if not self.is_log_level_valid(default_log_level) and self.is_log_level_valid(value):
+                print("The default_log_level is being set to a valid value '{value}'.  There should be no further action needed.".format(value=value),
```

Comment:
The only reason I put this is that when the user sets the value to a valid one, they initially get the invalid log level message.  I played with it a lot to see if I could prevent that from happening, but because the conf file is read before the value is changed, I thought the users should know everything is OK now.

Reply:
I see. Wouldn't be there a way to avoid that situation somehow, if not complicated?

Reply:
I would think so yes.  For now I just stated that the Log Level has been set to a valid value.  Down side is the message displays every the command is ran with a valid message.  This might be OK, but after I make the rest of the changes, I will take another look at this.